plugins/modules/aci_fabric_setup_pod.py

```diff
@@ -0,0 +1,292 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2023, Samita Bhattacharjee (@samitab) <samitab@cisco.com>
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+from ansible_collections.cisco.aci.plugins.module_utils import constants
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {"metadata_version": "1.1", "status": ["preview"], "supported_by": "certified"}
+
+DOCUMENTATION = r"""
+---
+module: aci_fabric_setup_pod
+short_description: Manage Fabric Setup Pod (fabric:SetupP)
+description:
+- Manage Fabric Setup Policy of a Pod on Cisco ACI fabrics.
+options:
+  podId:
+    description:
+    - The Pod identifier.
+    - Accepted value range between C(1) and C(254).
+    type: int
+    aliases: [ pod, id ]
+  podType:
+    description:
+    - The Pod type
+    type: str
+    choices: [ physical, virtual ]
+    default: physical
```

Comment:
Does apic default when not provided or is this default required?

Reply:
APIC defaults to physical. Should we not set a default if APIC already does?

Reply:
Correct, we then only document that apic defaults to a value. 

`- The APIC defaults to C(physical) when unset during creation.`